api/hydra/sources.ttl

```diff
@@ -70,3 +70,12 @@ dataCube:Column
                                                                  hydra:title    "Mapped" ;
                                                                  hydra:description
                                                                                 "Indicates that the column is used in a dimension or measure." ] .
+
+_:DeleteSource
+  a                  hydra:SupportedOperation, hydra-box:View, api:DeleteSource ;
```

Comment:
Should it also have `schema:DeleteAction`?

Reply:
Sure, I don't see why not